File: mobile/lib/repository/challenge.dart
class Challenge {
  int id;
  String title;
  String description;
  double latitude;
  double longitude;
  String imageUrl;
  String thumbnailUrl;

  Challenge({
    this.id,
    this.title,
    this.description,
    this.latitude,
    this.longitude,
    this.imageUrl = '',
    this.thumbnailUrl = '',
  });
}

File: web/app/Challenge.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Challenge extends Model
{
    protected $fillable = [
        'title', 'description', 'location_id'
    ];
    public function location(){
        return $this->belongsTo('App\Location');
    }
    public function user(){
        return $this->belongsTo('App\User');
    }
}
